data/seeds/018-service_entries.js

```diff
@@ -0,0 +1,164 @@
+const faker = require('faker/locale/en_US');
+
+const {
+  fakeServiceEntryIds,
+  getRandWithZero,
+  getRand,
+} = require('../seedHelpers');
+
+// need these for the faker and mapper
+const { service_type_programs } = require('./003-service_type_programs');
+const { recipients } = require('./015-recipients');
+const { providers } = require('./005-providers');
+const { locations } = require('./013-locations');
+
+const service_entries = fakeServiceEntryIds.map((id) => {
+  const providerNumToUse = getRandWithZero(providers.length);
+  const recipientNumToUse = getRandWithZero(recipients.length);
+  const stpNumToUse = getRand(service_type_programs.length);
+  const locationNumToUse = getRandWithZero(locations.length);
+  return {
+    service_entry_id: id,
+    primary_provider_id: providers[providerNumToUse].provider_id,
+    primary_recipient_id: recipients[recipientNumToUse].recipient_id,
+    service_type_program_id: stpNumToUse,
+    apply_service_to_household: faker.datatype.boolean(),
+    service_date: faker.date.past(3),
+    service_time: faker.time.recent('abbr'),
+    service_duration: null,
+    service_value: getRandWithZero(300),
+    service_quantity: null,
+    service_entry_notes: '',
+    // service_entry_data: {
+    //   custom: fakeServiceEntryTypeCustomData(serviceTypeNumToUse - 1), // In this case using '-1' since the fake service creator is zero indexed
+    // },
+    service_entry_data: {}, // keeping above for now, may get rid of it
```

Comment:
Will likely need to build out the structure to allow for multiple (non-primary) providers and/or recipients into this jsonb object at a later date.